apps/super-app/src/common/components/ServiceCard.tsx

```diff
@@ -16,7 +16,7 @@ export function ServiceCard(props: ServiceCardAbout) {
     <Card sx={{ display: 'flex', marginBottom: '16px' }}>
       <CardMedia
         component="img"
-        sx={{ width: '5rem', height: '5rem', margin: 'auto 16px' }}
+        sx={{ width: '3.5rem', height: '3.5rem', margin: 'auto 0 auto 16px' }}
```

Comment:
Hm, maybe at this point it would be clearer to specify `margin-[top|bottom|left|right]` separately, but I don't feel strongly about it.

Reply:
Hmmm, feels like it'll be too verbose. Shall we leave it as-is for now?

Reply:
IMO verbosity is less of a problem than understandability, but let's leave it.